config/artifacts.json

```diff
@@ -834,8 +834,7 @@
 	{
 		"bonuses" : [
 			{
-				"subtype" : 12,
-				"type" : "SECONDARY_SKILL_PREMY",
+				"type" : "UNDEAD_RAISE_CHANCE",

```

Comment:
No, it will not work always, because of 	```if (hasImprovedNecromancy)``` in new necromancy. Hero without skill or full assembled cloak will not have impoved necromancy.

Reply:
There are also:
- archery and eagle eye artifacts that should NOT work without skill
- resistance artifacts however should work without skill

But can't remember whether these arts should boost stat directly or whether they boost skill, e.g. does "+10% to Archery" artifact improves hero archery by 10% (so 5% to ranged damage with expert archery) or directly increases all ranged damage by 10%?

Reply:
@IvanSavenko Eagle Eye artifacts are fine, because they set only chance, not level.
Archery artifacts working without skill in develop too, AFAIK (they set SECONDARY_SKILL_PREMY with ARCHERY directly). Can add a check to avoid archery artifacts working.

Reply:
@IvanSavenko fixed archery artifacts, added another feature to Json.